### PR TITLE
[ADD][connector_voicent][ISSUE-186] Add basic module with models and …

### PR DESCRIPTION
**Pull request by YounessMaafi**
…views for module : connector_voicent

#186

### CODE REVIEW

**Comment by max3903** [comment]
@YounessMaafi Can you fix the Travis errors:
https://travis-ci.org/OCA/connector-telephony/jobs/535553479#L615

Let me know here if something is not clear.

**Comment by YounessMaafi** [comment]
Hi @max3903 
I guess since it's my first contribution, i'll need a little more time to learn how to contribute properly following the standards.
And setup the right plugins to my EDI to make the task easier.
However, please let me know if there is anyother Travis error that i should fix in priority.
Thanks,

**Comment by max3903** [comment]
@YounessMaafi Which IDE are you using? I am using PyCharm and it takes care of the PEP8 stuff. You can also install and run flake8.

File: connector_voicent/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    'license': 'AGPL-3',
+    'summary': """
+Voicent Connector
+=========
+Usage:
+    Go to Connectors > Backends > Voicent Backends
+    Create a new Voicent Backend with the host and port
+    Create Call Lines to determine when (which stage in the process) calls are added to the queue
+    Create Time Line to determine when (what time) calls are made
+""",
```
Comment (max3903) [inline connector_voicent/__manifest__.py]: @YounessMaafi Runbot is complaining about the summary being too long. You are limited to 256 characters.

Just put something like:
`'summary': 'Automate phone calls based on stage changes',`

File: connector_voicent/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    'data': [
+        # Security
+        'security/ir.model.access.csv',
+        # Views
+        'view/res_partner.xml',
+        'view/backend_voicent.xml',
+        ],
+    'demo': [],
+    'installable': True,
+    'auto_install': False,
```
Comment (max3903) [inline connector_voicent/__manifest__.py:29]: @YounessMaafi Please add:
```
'development_status': 'Stable',
'maintainers': [
    'younessmaafi',
    'max3903',
],
```

**Comment by YounessMaafi** [comment]
Hi @max3903 

Thanks a lot for your help,
I'm using VCS, and yes i'll make pylint and flake8 working, and i'll push the changes you just requested ( i guess they are related to the last 2 checks in Travis and Runbot ).
I'll be pushing them in seconds.

Are the other linting warnings not blocking ?

Thanks again,

**Comment by max3903** [comment]
@YounessMaafi They are not blocking. They come from other modules.

@murtuzasaleh will take it from here and complete the connector.

Thank you!

**Comment by murtuzasaleh** [comment]
Superseeded by #189

**Comment by murtuzasaleh** [comment]
@YounessMaafi Can you please give me your branch (12.0-connector-voicent-186) access. (Go to Your Branch << Settings << Collaborators << Add Collaborator (murtuzasaleh)
new_collaborators

@max3903

**Pull request closed without merging**